lib/eight_ball.rb

```diff
@@ -116,7 +116,7 @@ def self.with(name, parameters = {})
   # If the {EightBall::Providers Provider} does not expose a
   # {EightBall::Marshallers Marshaller}, this will default to the
   # {EightBall::Marshallers::Json JSON Marshaller}.
-  def self.marshall(marshaller = nil)
+  def self.marshall(marshaller = nil, features = EightBall.features)
```

Comment:
Should this be keyword params? Saves on having to do `marshall(nil, features)` to set the variable

Reply:
Discussed IRL - this would be better, yes, but would entail a breaking change. Future™️.